modules/core/src/lib/deck.ts

```diff
@@ -1007,9 +1009,13 @@ export default class Deck<ViewsT extends ViewOrViews = null> {
 
     this.deckPicker = new DeckPicker(this.device);
 
+    const parent = this.props.parent || document.body;
+    const widgetParent =
+      parent.querySelector<HTMLDivElement>('.deck-widgets-container') || this.canvas?.parentElement;
```

Comment:
Should this be `this.props.parent?.querySelector()` like the `eventRoot`? Searching under `document.body` is suspicious...